src/Consolonia.Core/Infrastructure/AnsiConsoleOutput.cs

```diff
@@ -0,0 +1,170 @@
+using System;
+using System.Text;
+using Avalonia;
+using Avalonia.Media;
+using Consolonia.Core.Drawing;
+using Consolonia.Core.Drawing.PixelBufferImplementation;
+using Consolonia.Core.Helpers;
+using Consolonia.Core.Text;
+
+namespace Consolonia.Core.Infrastructure
+{
+    /// <summary>
+    ///     Console implementation which uses ANSI escape sequences for output
+    /// </summary>
+    public class AnsiConsoleOutput : IConsoleOutput
+    {
+        private const string TestEmoji = "👨‍👩‍👧‍👦";
+        private PixelBufferCoordinate _headBufferPoint;
+
+        private bool? _supportEmoji;
+
+        public bool CaretVisible { get; private set; }
```

Comment:
I don't see CaretVisible property is used anywhere

Reply:
yeah, huh. I have removed.